app/kumactl/pkg/cmd/root_context.go

```diff
@@ -22,6 +26,7 @@ type RootRuntime struct {
 	Now                        func() time.Time
 	NewResourceStore           func(*config_proto.ControlPlaneCoordinates_ApiServer) (core_store.ResourceStore, error)
 	NewDataplaneOverviewClient func(*config_proto.ControlPlaneCoordinates_ApiServer) (kumactl_resources.DataplaneOverviewClient, error)
+	NewDpTokenClient           func(string) (tokens.DpTokenClient, error)
```

Comment:
Why `DpTokenClient` instead of `DataplaneTokenClient` ? It doesn't follow any naming guidelines